### PR TITLE
DR-1175 Fix button text, API url typo and api values sent

### PR DESCRIPTION
**Pull request by mvillaseco**
# Overview
Fix several issues

# Changes
 - Fix text in buy button
 - Fix API url type
 - Fix Values sent to API.

### CODE REVIEW

File: src/wwwroot/controllers/BillingCtrl.js [hunk 1]
```diff
@@ -137,8 +137,12 @@
       vm.viewExpDate = form.expDate.$viewValue;
     }
     function submitBillingPayment() {
-      var paymentData = { plan : planName, creditCard: { number: vm.cc.number, securityCode: vm.secCode.number, expiration: vm.expDate} };
-      return settings.billingPayment(paymentData)
+      var agreements = {
```
Comment (nreal) [inline src/wwwroot/controllers/BillingCtrl.js]: for some reason this name is plural? because  think that should be singular

**Comment by nreal** [comment]
👍

File: src/wwwroot/controllers/BillingCtrl.js (hunk 1 above)
Comment (mvillaseco) [inline src/wwwroot/controllers/BillingCtrl.js]: Yes, it should be agreements because it's an object of several items.

**Comment by andresmoschini** [comment]
:shipit:

**Pull request merged**